File: Python/1048 - Salary Increase.py
salary=float(input())
if 0<=salary<=400:
  money_earned= (salary * 15) / 100
  new_salary= salary + money_earned
  print(f"Novo salario: {new_salary:.2f}")
  print(f"Reajuste ganho: {money_earned:.2f}")
  print("Em percentual: 15 %")
elif 400.01<=salary<=800:
  money_earned=(salary * 12) /100
  new_salary= salary + money_earned
  print(f"Novo salario: {new_salary:.2f}")
  print(f"Reajuste ganho: {money_earned:.2f}")
  print("Em percentual: 12 %")
elif 800.01<=salary<=1200:
  money_earned=(salary * 10) /100
  new_salary= salary + money_earned
  print(f"Novo salario: {new_salary:.2f}")
  print(f"Reajuste ganho: {money_earned:.2f}")
  print("Em percentual: 10 %")
elif 1200.01<=salary<=2000:
  money_earned=(salary * 7) /100
  new_salary= salary + money_earned
  print(f"Novo salario: {new_salary:.2f}")
  print(f"Reajuste ganho: {money_earned:.2f}")
  print("Em percentual: 7 %")
elif 2000.01<=salary:
  money_earned=(salary * 4) /100
  new_salary= salary + money_earned
  print(f"Novo salario: {new_salary:.2f}")
  print(f"Reajuste ganho: {money_earned:.2f}")
  print("Em percentual: 4 %")

File: Java/1048 - Salary Increase.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double salario = sc.nextDouble();
        double reajuste = 0;
        int percentual = 0;

        if (salario > 0 && salario <= 400) {
            percentual = 15;
            reajuste = salario * 0.15;
        } else if (salario > 400 && salario <= 800) {
            percentual = 12;
            reajuste = salario * 0.12;
        } else if (salario > 800 && salario <= 1200) {
            percentual = 10;
            reajuste = salario * 0.10;
        } else if (salario > 1200 && salario <= 2000) {
            percentual = 7;
            reajuste = salario * 0.07;
        } else if (salario > 2000) {
            percentual = 4;
            reajuste = salario * 0.04;
        }

        double novoSalario = salario + reajuste;

        System.out.printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %d %%\n", novoSalario, reajuste, percentual);
    }
}